rules/xcode_sdk_frameworks.bzl

```diff
@@ -231,13 +244,15 @@ def _create_build_file_for_sdk(
     for i in range(module_count):
         name_idx = 2 * i
         details_idx = 2 * i + 1
-        targets.append(_target_for_module(
+        target = _target_for_module(
```

Comment:
`_target_for_module` might return `None`? is that even allowed?  or this rules is still in construction since some code is commented out